handbook/preview-features/preview-features.md

```diff
@@ -0,0 +1,35 @@
+# Preview Features
+
+This section contains guided help for preview features in the current SystemLink release. While preview features are not yet complete and have limited functionality, they provide an understanding of the direction the product is taking and give users the opportunity to experiment and give feedback.
+
+## Creating and Uploading Analysis Automation Procedures with Jupyter Notebooks
+
+Analysis Automation performs recurring analyses with similar data and exports the results as a PDF or graphics file for management reporting. Analysis automation procedures define the analyses and contain the analysis script, the parameter definition, and the search query for the respective data. Currently, you create procedures in DIAdem VBS or Python, but now you can also create them in Jupyter and upload them straight to Analyis Automation.  
+
+### Prerequisites
+
+To create analysis automation procedures with Jupyter in SystemLink, you need to install SystemLink 2021 R1 and the Measurement Data Analysis module.
+
+### Steps to Take
+
+Create an analysis automation procedure from the analysis notebook you write in Jupyter and upload it to the procedures library in Analysis Automation.
+
```

Comment:
Are there screenshots that would be helpful for this section?

Reply:
added screenshot